src/parser/warrior/arms/modules/Abilities.js

```diff
@@ -75,7 +83,7 @@ class Abilities extends CoreAbilities {
         },
         castEfficiency: {
           suggestion: true,
-          recommendedEfficiency: 0.6, // I don't know at what efficiency this talent becomes worth it so I'm keeping it save
+          recommendedEfficiency: 0.9, // I don't know at what efficiency this talent becomes worth it so I'm keeping it save
         },
         enabled: combatant.hasTalent(SPELLS.WARBREAKER_TALENT.id), // replaces Colussus Smash
```

Comment:
Is that comment still relevant since you upped cast efficiency drastically?

Reply:
I removed this comment since it is now irrelevant